### DISCUSSION TITLE
小白求教网页的“不安全连接”是否有影响

### DISCUSSION THREAD

**Discussion by Suntxx** [MoeNetwork/Tieba-Cloud-Sign#194] [疑问] [answered]
如图就是连接栏这个锁提示的不安全，看着很别扭，有影响吗或者可以设置成安全状态吗，小白求教大佬
![image](https://user-images.githubusercontent.com/113056239/188950953-9c989953-1ab4-468d-a1be-53e1f75804f0.png)

**Comment by Suntxx** [comment]
一切功能可以用非常好用，就是这个不安全看着怪别扭

**Comment by BANKA2017** [comment, marked as answer]
chrome会对http连接的页面提醒不安全，我看这个8080的端口号应该是用docker在本地的nas或者路由器部署的吧，一般都不会有危险

**Reply by Suntxx** [reply]
好的，感谢大佬，之前都是跟着别人视频一步一步搭的，有一个类似网站也是显示不安全，但是跟他设置后可以设置成锁上的状态，但是也不懂具体原理，后来又搭过一个qiandao的站也是显示不安全，当然用了很久了倒也没什么事，就是每回上去看见不安全就感觉不舒服😄

**Reply by Suntxx** [reply]
对了大佬是在服务器搭建的

**Reply by BANKA2017** [reply]
有域名可以研究绑定域名以及https
没有域名又不打算绑定域名的可以考虑改成高位端口避免被扫常见端口

一般来说是没什么大问题的，但不能保证永远没有问题

**Reply by Suntxx** [reply]
原来是这样啊，突然想起来之前搭的winxrary是nginx绑的阿里云域名，估计是这样那个网站才能上锁，这就去研究下怎么把域名给他绑上

**Reply by Suntxx** [reply]
大佬能指点下这个操作关键词吗，我好去搜个教学视频..

**Reply by Suntxx** [reply]
大佬我研究了一圈，有点疑问希望大佬能帮忙解答下，大概了解到不安全的原因是因为现在是网站是http的要转为https，需要给服务器安装ssl证书，我想问下我这个服务器上面一共有三个项目，其中有1个翻墙是通过nginx绑了域名是安全的已经是https了，还有2个签到网站都是http显示不安全的，我现在如果给服务器安装ssl证书的话不会影响到已经绑过域名的翻墙吧

**Reply by BANKA2017** [reply]
不会有影响

**Reply by n0099** [reply]
请您使用[certbot](https://certbot.eff.org)每三个月申请一次[let's encrypt](letsencrypt.org)的证书，然后修改nginx conf来使用这个证书

**Reply by n0099** [reply]
> 好的，感谢大佬，之前都是跟着别人视频一步一步搭的，有一个类似网站也是显示不安全，但是跟他设置后可以设置成锁上的状态，但是也不懂具体原理，后来又搭过一个qiandao的站也是显示不安全，当然用了很久了倒也没什么事，就是每回上去看见不安全就感觉不舒服smile

谁的视频
